portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,39 +34,70 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+ /** 
+  * This class represents CommentDescription.
+  */
+  public class UserComment {
+      private String emailId;
+      private String comment;
+      UserComment(String emailId,String comment) {
```

Comment:
One line after the field declarations. Also one space after comma.

```suggestion

      UserComment(String emailId, String comment) {
```